docs/examples/multi-uv.c

```diff
@@ -129,14 +129,12 @@ static void check_multi_info(void)
   }
 }
 
-void curl_perform(uv_poll_t *req, int status, int events)
+static void curl_perform(uv_poll_t *req, int status, int events)
 {
   int running_handles;
   int flags = 0;
   curl_context_t *context;
 
-  uv_timer_stop(&timeout);
```

Comment:
I think so. The other example programs for `CURLMOPT_SOCKETFUNCTION` do not reset the timer in the socket callback function. (asiohiper.cpp, evhiperfifo.c, ghiper.c, hiperfifo.c)